src/styles/main.scss

```diff
@@ -1,6 +1,10 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+// ------------
+// IMPORTANT:
+// PLEASE NOTE -- THIS FILE IS NOT IN USE ANYMORE
```

Comment:
If it's not in use, let's just delete it entirely.

Reply:
Done.